scripts/scons_plugin/config.py

```diff
@@ -52,6 +56,38 @@ def linux_dirs():
         if os.path.isdir(llvmdir):
             return llvmdir
 
+# Gets `PKG_CONFIG_PATH` from environment `env` and tries to append `add_prefix`/lib/pkgconfig.
+# Tolerant to absense of any or both of those two. If both are absent, returns empty string.
+def _compose_pkg_config_path(env, add_prefix, is_doc_test=False):
+    """
+    >>> _compose_pkg_config_path({'PKG_CONFIG_PATH': '/a/b'}, '/usr/local/qqq', True)
+    '/a/b:/usr/local/qqq/lib/pkgconfig'
+    >>> _compose_pkg_config_path({}, '/usr/local/qqq', True)
+    '/usr/local/qqq/lib/pkgconfig'
+    >>> _compose_pkg_config_path({'PKG_CONFIG_PATH': '/a/b/'}, None, True)
+    '/a/b/'
+    >>> _compose_pkg_config_path({}, None, True)
+    ''
```

Comment:
I suggest removing these tests to avoid maintaining two versions of code (test vs non-test). Discussion in chat.